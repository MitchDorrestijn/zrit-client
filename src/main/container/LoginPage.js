import React, {Component} from 'react';
import {Container, Row, Col} from 'reactstrap';

import LoginImg from '../components/LoginImg';
import LoginForm from "../components/LoginForm";

import 'bootstrap/dist/css/bootstrap.css';
import '../css/login.css';

//this class takes care of rendering of the LoginPage
export default class LoginPage extends Component {
    render() {
        return (
            <Container>
                <Row>
                    <Col md={6}>
                        <LoginImg/>
                    </Col>

                    <Col md={6}>
                        <LoginForm/>
                    </Col>
                </Row>
            </Container>
        );
    }
}