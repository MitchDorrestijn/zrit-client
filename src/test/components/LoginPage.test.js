import React from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import LoginPage from '../../main/container/LoginPage';

const wrapper = shallow(<LoginPage />);
test('LoginPage component should match snapshot', () => {
    expect(toJSON(wrapper)).toMatchSnapshot();
});

test('Should render one Container component', () => {
    expect(wrapper.find('Container').length).toBe(1);
});

test('Should render one Row component', () => {
    expect(wrapper.find('Row').length).toBe(1);
});

test('Should render two Col component', () => {
    expect(wrapper.find('Col').length).toBe(2);
});
