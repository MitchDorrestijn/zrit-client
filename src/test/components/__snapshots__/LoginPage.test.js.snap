// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage component should match snapshot 1`] = `
<Container
  tag="div"
>
  <Row
    tag="div"
  >
    <Col
      md={6}
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <LoginImg />
    </Col>
    <Col
      md={6}
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <LoginForm />
    </Col>
  </Row>
</Container>
`;
